migtests/scripts/functions.sh

```diff
@@ -123,11 +123,17 @@ grant_user_permission_oracle(){
 	END;
 	/
 	BEGIN
-    	FOR R IN (SELECT owner, object_name FROM all_objects WHERE owner=UPPER('${db_schema}') and object_type in ('VIEW','SEQUENCE','TABLE PARTITION','TABLE','SYNONYM','MATERIALIZED VIEW')) LOOP
+    	FOR R IN (SELECT owner, object_name FROM all_objects WHERE owner=UPPER('${db_schema}') and object_type in ('VIEW','SEQUENCE','TABLE PARTITION','SYNONYM','MATERIALIZED VIEW')) LOOP
         	EXECUTE IMMEDIATE 'grant select on '||R.owner||'."'||R.object_name||'" to ybvoyager';
   		END LOOP;
 	END;
 	/
+	BEGIN
+		FOR R IN (SELECT owner, object_name FROM all_objects WHERE owner=UPPER('${db_schema}' ) and object_type ='TABLE' MINUS SELECT owner, table_name from all_nested_tables where owner = UPPER('${db_schema}')) LOOP
```

Comment:
Very minor nit: Can we please remove the extra space in `UPPER('${db_schema}' )`. Our official docs have this extra space as well. Can be updated too when this gets added.